### DISCUSSION TITLE
Improve the aspect ratio of preview in skin layout editor

### DISCUSSION THREAD

**Discussion by expiteRz** [ppy/osu#17407] [Ideas] [answered]
The width of preview area is shrunk in skin layout editor, and it's slightly inconvenience to edit layout for some skins. Could you please make the aspect ratio can be kept in it?
![osu_skin_layout_discussion](https://user-images.githubusercontent.com/51064411/159460919-96b493c6-43d0-43a4-a740-338fe131e091.jpg)
.

**Comment by peppy** [comment, marked as answer]
The intention is that you make skins which scale to any aspect ratio, using anchors and origins. But the eventual goal is to allow locking aspect ratio for those that want to. It requires a bit more thought to how the cross-component interactions are done.
